requirements-test.txt

```diff
@@ -1,5 +1,12 @@
-pytest
-pytest-cov
 dateparser
+dtw==1.3.3
+matplotlib
 mock
-sklearn
```

Comment:
why change ? https://stackoverflow.com/questions/38733220/difference-between-scikit-learn-and-sklearn

Reply:
To install `scikit-learn` using pip we have to run ```pip install scikit-learn```. That is why I changed it in the requirements-test.txt (assuming that package will be installed during testing)

https://scikit-learn.org/0.16/install.html

To import `scikit-learn` the code is ```import sklearn```

Reply:
`sklearn`'s an alias in PyPI for "latest version of scikit-learn" so version pinning doesn't work with it. Also, the alias does not exist in conda. Neither issue matters here in particular since this file is just for our unit tests, but in general `scikit-learn` is the better all-around name to use

Reply:
Right, I just tried ```pip install sklearn```, it seems to work. However, I clearly remember doing ```pip install sklearn==<>``` that did not work.